test/unit/tech/flash.test.js

```diff
@@ -177,3 +186,28 @@ test('canHandleSource should be able to work with src objects without a type', f
   equal('maybe', canHandleSource({src: 'test.video.flv'}), 'should guess that it is a flash video');
   equal('', canHandleSource({src: 'test.video.wgg'}), 'should return empty string if it can not play the video');
 });
+
+test('seekable should be for the length of the loaded video', function() {
+  let duration = 23;
+
+  // mock out duration
+  tech.el().vjs_getProperty = function(name) {
```

Comment:
@heff The 4.x tests worked this way without causing the issues we're seeing in 5.x so I feel like we missed a test-infrastructure upgrade of some sort along the way. That said, I can't object too much if everything is passing now.